builtin/providers/aws/resource_aws_vpn_gateway.go

```diff
@@ -270,8 +270,8 @@ func vpnGatewayStateRefreshFunc(conn *ec2.EC2, id string) resource.StateRefreshF
 			return nil, "", nil
 		}
 
-		vpnGateway := &resp.VPNGateways[0]
-		return vpnGateway, *vpnGateway.State, nil
+		vpnGateway := resp.VPNGateways[0]
+		return vpnGateway, "", nil
```

Comment:
We were returning it, but never using it. It's kind of a funky remnant the original contributor got from copying LC. I should just overhaul the whole method...